basic/if_condition/question/1_positive_or_negative.py

```diff
@@ -10,19 +10,19 @@ def positive_or_negative(num):
 import unittest
 
 class Test(unittest.TestCase):
-    def test_positive_or_negative(self):
+    def test_positive_or_negative1(self):
         num = 5
         actual = positive_or_negative(num)
         expected = "POSITIVE"
         self.assertEqual(actual, expected)
     def test_positive_or_negative(self):
         num = -2
-        actual = positive_or_negative(num)
+        actual = positive_or_negative2(num)
```

Comment:
You have changed the actual function name instead of changing test_positive_or_negative